suite-native/transactions/src/components/TransactionDetail/TransactionDetailValuesSheet.tsx

```diff
@@ -35,7 +35,7 @@ const TodayHeaderCell = ({
     network,
     historicalRates,
 }: TodayHeaderCellProps) => {
-    const coins = useSelector(selectCoins);
+    const coins = useSelector(selectCoinsLegacy);
     const fiatCurrency = useSelector(selectFiatCurrency);
     const currentRates = coins.find(coin => coin.symbol === network)?.current?.rates;
```

Comment:
Shouldn't be `selectFiatRatesByFiatRateKey` selector used now?

Reply:
Should but that will require more refactoring that I want to do in this PR